internal/provider/provider.go

```diff
@@ -66,7 +66,6 @@ func New(version string) func() *schema.Provider {
 			},
 			ResourcesMap: map[string]*schema.Resource{
 				"juju_application":  resourceApplication(),
-				"juju_access_model": resourceAccessModel(),
```

Comment:
run `go fmt` on this file - i got a change in spacing after running `make install`